README.md

```diff
@@ -1 +1,34 @@
-# HelloMicroverseProject
\ No newline at end of file
+# HelloMicroverseProject
+
+> This is the first project at Microverse first week activity.
+
```

Comment:
- [ ] Kindly, Add a descriptive README file to your project, it will be great to add instructions on how to run the project. This includes `npm install` , `npm start` etc

- [ ] Your README looks great, good job 👍🏻  however  it will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser.. This includes `git clone URL` , `cd project_name` etc